docs/UserGuide.md

```diff
@@ -891,6 +892,15 @@ step-by-step command for performance and all-in-one command for event?
 
 ---
 
+**Q**: Why do I see a bunch of events and/or student lists added when I launch 
+the program?
```

Comment:
program or application? need to standardize throught UG.

Reply:
Ahh I'll change to application.